tests/Integration/Behaviour/Features/Scenario/Order/order_specific_prices.feature

```diff
@@ -57,7 +57,7 @@ Feature: Order from Back Office (BO)
       | total_shipping_tax_excl  | 7.0    |
       | total_shipping_tax_incl  | 7.42   |
     Then product "Test Product With Specific Price" in order "bo_order1" should have no specific price
-    Given I generate invoice for "bo_order1" order
+    Given I update order "bo_order1" status to "Payment accepted"
     And order "bo_order1" should have 1 invoices
```

Comment:
```suggestion
    And order "bo_order1" should have 1 invoice
```

Reply:
Not really the point of this PR ^^ And these plural sentences have been fixed in the 177 branch already so it will we there in the next merge